### DISCUSSION TITLE
Is there any reason why the deepin group in the Arch Repos contains deepin-wm and not deepin-kwin?

### DISCUSSION THREAD

**Discussion by weiameili** [linuxdeepin/developer-center#2990] [General | 常规讨论]
Hi, recently installed DDE on Arch linux, and it was full of bugs to an extent that it was completely unusable. Just saw that deepin-wm has been deprecated (but is still in the deepin group in the Arch package repos) and replaced it with the deepin-kwin package, after which everything worked fine.

Is there any specific reason why this package has not replaced the deepin-wm package in the deepin group in Arch repos?

**Comment by felixonmars** [comment]
@weiameili deepin-kwin completely breaks every time when kwin upgrades. There is no process upstream to help with the situation so that I still put deepin-wm as the default as it still basically works. @justforlxz is working on the compatibility issues caused by deepin-wm.
